apps/makescene/makescene.cc

```diff
@@ -800,30 +846,32 @@ is_noah_bundler_format (AppSettings const& conf)
 bool
 is_colmap_sfm_bundle_format (AppSettings const& conf)
 {
-    std::string cameras_txt_filename = util::fs::join_path(conf.input_path, 
-        "cameras.txt");
-    std::string cameras_bin_filename = util::fs::join_path(conf.input_path, 
-        "cameras.bin");
+    using util::fs::join_path;
+    std::string workspace_path = conf.input_path;
+    std::string model_path(join_path(workspace_path, "sparse"));
```

Comment:
Prefer assignment instead of constructor. It's equivalent, but semantically clearer. It's unheard of that people initialize their primitives with the value constructor `int x(10)` for the same reason.